web/html/src/manager/state/recurring-states.tsx

```diff
@@ -204,17 +206,22 @@ class RecurringStates extends React.Component<Props, State> {
   };
 
   render() {
-    const messages = this.state.messages ? <Messages items={this.state.messages} /> : null;
-    const notification = (
-      <Messages
-        items={[
-          {
-            severity: "warning",
-            text: "The timezone displayed is the server timezone. The scheduled time will be the server time.",
-          },
-        ]}
-      />
-    );
+    const messages = this.state.messages ? <Messages key="state-messages" items={this.state.messages} /> : null;
+    const notification =
+      localizedMoment.userTimeZone !== localizedMoment.serverTimeZone ? (
+        <Messages
+          key="notification-messages"
+          items={[
+            {
+              severity: "warning",
+              text: t(
+                "The below times are displayed is the server time zone {0}. The scheduled time will be the server time.",
```

Comment:
Nit: is -> in

Reply:
D'oh, thanks. 😄 Somehow I took "the server is in" and "times are in" and turned it into "the times is".